### ISSUE TITLE
PHP Code is broken after formatting.

### ISSUE THREAD

**Issue by juliusdev** [prettier/plugin-php#1758]
**Prettier 2.3.1**

**PHP Plugin 0.17.1**

```sh
      arrowParens: avoid
      # jsx and tsx rules
      jsxBracketSameLine: false
      printWidth: 200
      tabWidth: 2
      useTabs: false
      singleQuote: true
      phpVersion: "7.2"
      braceStyle: "1tbs"
      endOfLine: "auto"
```

**Input:**

```php
<?php // P1T_SPECIALITE_03
					echo Input::selectList('contratPDRControl.contratPDR'.$id.'.specialites', 
							array ('typeSelect' => 'SpecialitePDR', 
							'multiple' => true,
							'horizontal' => true,
							/*'texte' => 'Spécialité',*/
							'select' => 'contratPDRControl.selectedSpecialite',
							'methode' => 'contratPDRControl.changeSelectedSpecialites',
							'reRender' => $reRenderAllTarif,
							'class' => 'mb-3'
							), $controleur->mapDonneesReference['PXX_PDR_SPECIALITE'], true, false, false); ?>

```

**Output:**

```php
<?php // P1T_SPECIALITE_03
// P1T_SPECIALITE_03
?>echo Input::selectList(
       'contratPDRControl.contratPDR' . $id . '.specialites',
       [
         'typeSelect' => 'SpecialitePDR',
         'multiple' => true,
         'horizontal' => true,
         /*'texte' => 'Spécialité',*/
         'select' => 'contratPDRControl.selectedSpecialite',
         'methode' => 'contratPDRControl.changeSelectedSpecialites',
         'reRender' => $reRenderAllTarif,
         'class' => 'mb-3',
       ],
       $controleur->mapDonneesReference['PXX_PDR_SPECIALITE'],
       true,
       false,
       false
     ); ?>
```

**Expected behavior:**
Here the php code is broken : the ``<?php`` is closed just before the ``echo``.

**Comment by juliusdev** [comment]
Another exemple :

Input:

```php
<?php //FOO
					echo "foo";?>
					
        <?php //BAR
        echo "bar";?> 
```

Output:

```php
<?php //FOO
echo 'foo'; ?>
					
        <?php //BAR
//BAR
?>echo 'bar';
?>
```

Here the problem is correlated  with the comment just after `<?php`.

**Comment by czosel** [comment]
Fixed in `v0.17.5` :tada:
